client/src/templates/layouts/NavigationLayout.js

```diff
@@ -111,11 +139,236 @@ const NavigationLayout = (props) => {
   const history = useHistory();
 
   const [drawerOpened, setDrawerOpened] = useState(false);
+  const [changeValue, setChangeValue] = useState(false);
```

Comment:
You did not need to to use this. All of the form state handling is being done in `feedbackState`. Creating this was an extra step that doesn't fit in to what is going on here. And from what I can see it is not being used to send with the request.

Reply:
The main use of this is to enable the radio buttons. I'm not sure what you mean by feedbackState?

Reply:
It is the state variable used to collect all the data and pass it to the API call. Look at line 143 and 164.